docs/use_cases/model_applications/s2s_mid_lat/UserScript_obsERA_obsOnly_Blocking.py

```diff
@@ -1,173 +1,209 @@
 #
-# METplus sets environment variables based on the values in the METplus configuration file.
-# These variables are referenced in the MET configuration file. **YOU SHOULD NOT SET ANY OF THESE ENVIRONMENT VARIABLES YOURSELF! THEY WILL BE OVERWRITTEN BY METPLUS WHEN IT CALLS THE MET TOOLS!** If there is a setting in the MET configuration file that is not controlled by an environment variable, you can add additional environment variables to be set only within the METplus environment using the [user_env_vars] section of the METplus configuration files. See the 'User Defined Config' section on the 'System Configuration' page of the METplus User's Guide for more information.
-#
-# See the following files for more information about the environment variables set in this configuration file.
-#
-# parm/use_cases/met_tool_wrapper/RegridDataPlane/RegridDataPlane.py
-# parm/use_cases/met_tool_wrapper/PCPCombine/PCPCombine_derive.py
-# parm/use_cases/met_tool_wrapper/PCPCombine/PCPCombine_subtract.py
+# This case does not use MET configuration files.
 
 ##############################################################################
-# Python Scripts
+# Python Embedding
 # ----------------
 #
-# This use case uses Python scripts to perform the blocking calculation
+# This use case does not use python embedding
+
+##############################################################################
+# User Scripting
+# --------------
 #
-# parm/use_cases/model_applications/s2s_mid_lat/UserScript_obsERA_obsOnly_Blocking/Blocking_driver.py:
-# This script calls the requested steps in the blocking analysis for a forecast, observation, or both.
+# This use case runs the blocking driver.  The blocking driver runs the user selected steps
+# of the blocking calculation.  These steps are specified in OBS_STEPS in the [user_env_vars]
+# section fo the configuration file in the following format:
 #
-# metcalcpy/contributed/blocking_weather_regime/Blocking.py:
-# This script runs the requested steps, containing the code for computing CBLs, computing IBLs, computing GIBLs,
-# and computing blocks.  See the METcalcpy `Blocking Calculation Script <https://github.com/dtcenter/METcalcpy/blob/develop/metcalcpy/contributed/blocking_weather_regime/Blocking.py>`_ for more information.
+#  | OBS_STEPS = CBL+PLOTCBL+IBL+PLOTIBL+GILB+CALCBLOCKS+PLOTBLOCKS
 #
-# metcalcpy/contributed/blocking_weather_regime/Blocking_WeatherRegime_util.py:
-# This script contains functions used by both the blocking anwd weather regime analysis, including the code for 
-# determining which steps the user wants to run, and finding and reading the input files in the format from the output
-# pre-processing steps.  See the METcalcpy `Utility script <https://github.com/dtcenter/METcalcpy/blob/develop/metcalcpy/contributed/blocking_weather_regime/Blocking_WeatherRegime_util.py>`_ for more information.
+# The possible steps are computing the CBLs or central blocking latitude (CBL), plotting CBLs 
```

Comment:
```suggestion
# The possible steps are computing the CBLs or Central Blocking Latitude (CBL), plotting CBLs 
```
I am suggesting capitalization to be consistent with the capitalization in the Scientific Objective section.